src/main.js

```diff
@@ -54,11 +63,27 @@ function switchView(hiddenPage, visiblePage, hiddenButton1, hiddenButton2, visib
   hiddenButton2.classList.toggle('hidden')
   visibleButton.classList.toggle('hidden')
 }
-switchView()
+// switchView()
 
 function viewHome() {
   homeView.classList.toggle('hidden')
   homeButton.classList.toggle('hidden')
   randomCoverButton.classList.toggle('hidden')
   saveNewButton.classList.toggle('hidden')
 }
+
+function saveMyBook(event) {
```

Comment:
This function looks good! Good call pulling in the variables that will display on the home page.